newrelic-agent/src/main/java/com/newrelic/api/agent/NewRelicApiImplementation.java

```diff
@@ -298,6 +298,7 @@ public void addCustomParameters(Map<String, Object> params) {
      */
     @Override
     public void setUserId(String userId) {
+        MetricNames.recordApiSupportabilityMetric(MetricNames.SUPPORTABILITY_API_SET_USER_ID);
```

Comment:
I don't know the specifics of the ticket, but do we want to send the metric if the call is ignored due to high security being enabled? Or have another metric for when it _is_ ignored due to high security?